2-conan-build.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/env bash
```

Comment:
Run in Debian 10 container:
```
root@ef5af972deea:~/conan/synfig# ./1-conan-export.sh 
bash: ./1-conan-export.sh: /bin/env: bad interpreter: No such file or directory
```

Same on MacOS. I suggest to use `/usr/bin/env` as a more portable solution.